[Bug] Example `enhanced_websocket_transactions.rs` fails to compile: `TransactionSubscribeFilter::standard` type mismatch

---

**Bug: Type mismatch in `TransactionSubscribeFilter::standard` example prevents compilation (expects `&Pubkey` but gets `&Address`)**

#### Summary
The example code in `examples/enhanced_websocket_transactions.rs` (on the `dev` branch) does not compile out-of-the-box due to a type mismatch in the `RpcTransactionsConfig` setup. Specifically, `TransactionSubscribeFilter::standard(&key)` expects a `&solana_pubkey::Pubkey` but receives a `&Address` (from Helius types). This leads to an `E0308` compilation error.

This issue could confuse new users trying to follow the example, impacting the first-time developer experience with the SDK. A quick fix in the example (e.g., adjusting the key type or import) would make it plug-and-play.

#### Environment
- **Helius SDK version**: 0.3.2
- **Solana SDK version**: 3.0.0 (from `solana-sdk = "3.0.0"`)
- **Rust edition**: 2024
- **OS**: Debian GNU/Linux 12 (bookworm)
- **Cargo version**: cargo 1.90.0 (840b83a10 2025-07-30)
- **Branch**: dev (example file location)

#### Relevant Dependencies (Cargo.toml)
```
[package]
name = "heliusTest"
version = "0.1.0"
edition = "2024"

[dependencies]
helius = "0.3.2"
solana-sdk = "3.0.0"
tokio = "1.48.0"
tokio-stream = "0.1.17"
```

#### Steps to Reproduce
1. Clone the repo: `git clone https://github.com/helius-labs/helius-rust-sdk.git && cd helius-rust-sdk`
2. Create a new Cargo project with the above `Cargo.toml`.
3. Copy the example code into `src/main.rs` (see below for the exact code used).
4. Replace `"your_api_key"` with a valid Helius API key (though it fails at compile time).
5. Run `cargo build`.

#### Code (src/main.rs)
```rust
use helius::error::Result;
use helius::types::{Cluster, RpcTransactionsConfig, TransactionSubscribeFilter, TransactionSubscribeOptions};
use helius::Helius;
use solana_sdk::pubkey;
use tokio_stream::StreamExt;

#[tokio::main]
async fn main() -> Result<()> {
    let api_key: &str = "your_api_key";
    let cluster: Cluster = Cluster::MainnetBeta;

    // Uses custom ping-pong timeouts to ping every 15s and timeout after 45s of no pong
    let helius: Helius = Helius::new_with_ws_with_timeouts(api_key, cluster, Some(15), Some(45)).await?;

    let key: pubkey::Pubkey = pubkey!("BtsmiEEvnSuUnKxqXj2PZRYpPJAc7C34mGz8gtJ1DAaH");

    let config: RpcTransactionsConfig = RpcTransactionsConfig {
        filter: TransactionSubscribeFilter::standard(&key),
        options: TransactionSubscribeOptions::default(),
    };

    if let Some(ws) = helius.ws() {
        let (mut stream, _unsub) = ws.transaction_subscribe(config).await?;
        while let Some(event) = stream.next().await {
            println!("{:#?}", event);
        }
    }

    Ok(())
}
```

#### Expected Behavior
The code should compile successfully and run, subscribing to enhanced websocket transactions for the given pubkey (as described in the example).

#### Actual Behavior
Compilation fails with the following error:

```
error[E0308]: mismatched types
  --> src/main.rs:18:54
   |
18 | filter: TransactionSubscribeFilter::standard(&key),
   | ------------------------------------ ^^^^ expected `&Pubkey`, found `&__Pubkey`
   | |
   | arguments to this function are incorrect
   |
   = note: expected reference `&solana_pubkey::Pubkey`
              found reference `&Address`
note: associated function defined here
  --> /home/quark/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/helius-0.3.2/src/types/enhanced_websocket.rs:23:12
   |
23 | pub fn standard(key: &Pubkey) -> Self {
   | ^^^^^^^^
For more information about this error, try `rustc --explain E0308`.
error: could not compile `heliusTest` (bin "heliusTest") due to 1 previous error
```

#### Additional Context
- This was tested on November 8, 2025.
- The example appears to be directly from the `dev` branch at https://github.com/helius-labs/helius-rust-sdk/blob/dev/examples/enhanced_websocket_transactions.rs.
- Happy to provide a PR if needed!

---

--- a/examples/enhanced_websocket_transactions.rs
+++ b/examples/enhanced_websocket_transactions.rs
@@ -12,9 +12,9 @@ async fn main() -> Result<()> {
     // Uses custom ping-pong timeouts to ping every 15s and timeout after 45s of no pong
     let helius: Helius = Helius::new_with_ws_with_timeouts(api_key, cluster, Some(15), Some(45)).await?;
 
-    let key = pubkey!("BtsmiEEvnSuUnKxqXj2PZRYpPJAc7C34mGz8gtJ1DAaH");
+    let key: pubkey::Pubkey = pubkey!("BtsmiEEvnSuUnKxqXj2PZRYpPJAc7C34mGz8gtJ1DAaH");
 
-    let config = RpcTransactionsConfig {
+    let config: RpcTransactionsConfig = RpcTransactionsConfig {
         filter: TransactionSubscribeFilter {
             account_include: Some(vec![key.to_string()]),
             vote: Some(false),

